examples/.nickel/stack.ncl

```diff
@@ -61,13 +61,13 @@ in
                 version = V.hurl,
                 hurlDir = "./bootstrap",
                 variables = {
-                  HURL_KEYCLOAK_BASE_URL = "http://external-keycloak-%{args.name}:8080",
-                  HURL_KEYCLOAK_ADMIN_USER = "admin",
-                  HURL_KEYCLOAK_ADMIN_PASSWORD = "admin",
-                  HURL_KEYCLOAK_REALM = args.realm,
-                  HURL_ALICE_USERNAME = "alice",
-                  HURL_ALICE_PASSWORD = "1234",
-                  HURL_ALICE_WALLET_CLIENT_ID = "alice-wallet",
+                  HURL_keycloak_base_url = "http://external-keycloak-%{args.name}:8080",
```

Comment:
Why we cannot switch to the uppercase format?

Reply:
I just learned the case needs to match the one in the script (lowercase convention). We can use the uppercase convention , we'll have to update the script and variable file as well.